playground/NodeEditorLib.js

```diff
@@ -0,0 +1,134 @@
+import { NodePrototypeEditor } from './editors/NodePrototypeEditor.js';
+import { ScriptableEditor } from './editors/ScriptableEditor.js';
+import { BasicMaterialEditor } from './editors/BasicMaterialEditor.js';
+import { StandardMaterialEditor } from './editors/StandardMaterialEditor.js';
+import { PointsMaterialEditor } from './editors/PointsMaterialEditor.js';
+import { FloatEditor } from './editors/FloatEditor.js';
+import { Vector2Editor } from './editors/Vector2Editor.js';
+import { Vector3Editor } from './editors/Vector3Editor.js';
+import { Vector4Editor } from './editors/Vector4Editor.js';
+import { SliderEditor } from './editors/SliderEditor.js';
+import { ColorEditor } from './editors/ColorEditor.js';
+import { TextureEditor } from './editors/TextureEditor.js';
+import { UVEditor } from './editors/UVEditor.js';
+import { PreviewEditor } from './editors/PreviewEditor.js';
+import { TimerEditor } from './editors/TimerEditor.js';
+import { SplitEditor } from './editors/SplitEditor.js';
+import { JoinEditor } from './editors/JoinEditor.js';
+import { StringEditor } from './editors/StringEditor.js';
+import { FileEditor } from './editors/FileEditor.js';
+import { CustomNodeEditor } from './editors/CustomNodeEditor.js';
+
+export const ClassLib = {
+	BasicMaterialEditor,
+	StandardMaterialEditor,
+	PointsMaterialEditor,
+	FloatEditor,
+	Vector2Editor,
+	Vector3Editor,
+	Vector4Editor,
+	SliderEditor,
+	ColorEditor,
+	TextureEditor,
+	UVEditor,
+	TimerEditor,
+	SplitEditor,
+	JoinEditor,
+	StringEditor,
+	FileEditor,
+	ScriptableEditor,
+	PreviewEditor,
+	NodePrototypeEditor
+};
+
+let nodeList = null;
+
+export const getNodeList = async () => {
+
+	if ( nodeList === null ) {
+
+		const response = await fetch( './Nodes.json' );
+		nodeList = await response.json();
+
+	}
+
+	return nodeList;
+
+}
```

Comment:
Here (and in the latter function) should be a semicolon after it (because it is an arrow function so a statement).